Gemfile.lock

```diff
@@ -311,8 +309,8 @@ GEM
       net-ssh (>= 2.8.0)
     sshkit-interactive (0.3.0)
       sshkit (~> 1.12)
-    standard (0.12.0)
-      rubocop (= 1.8.1)
+    standard (0.13.0)
```

Comment:
### [standard](https://github.com/testdouble/standard)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 0.12.0 → 0.13.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **9** commits. See the full changes on [the compare page](https://github.com/testdouble/standard/compare/v0.12.0...v0.13.0).

These are the individual commits:
- (a830f99) [Add support for --stderr](https://github.com/testdouble/standard/commit/a830f9945ff609e7ce942df5e6931f5ca240e5b3)
- (d1d9aa4) [Merge pull request #256 from knu/support_--stderr](https://github.com/testdouble/standard/commit/d1d9aa4cee197b8753c6711e1aca9b1da37cd9e2)
- (38ff4ff) [Bump pry from 0.13.1 to 0.14.0](https://github.com/testdouble/standard/commit/38ff4ff54f372f98321f017cc5ea7e15d1aafbbd)
- (f731510) [Merge pull request #257 from testdouble/dependabot/bundler/p…](https://github.com/testdouble/standard/commit/f73151049242cf627437e601d8953db7980ea5e6)
- (e5ed823) [update to 1.10.0 and also enable 3 new rules from past relea…](https://github.com/testdouble/standard/commit/e5ed8235d8a3f646a720645a34c9a4562f24db7c)
- (07ae731) [fix typo](https://github.com/testdouble/standard/commit/07ae731f3fceeb174b5f3c179da3a3bdcd6d5e93)
- (321112a) [2.4 test should use 2.5 not 2.7](https://github.com/testdouble/standard/commit/321112a706e27775d8eb4d40e290be59da871ef8)
- (479a048) [Merge pull request #258 from testdouble/update-standard-to-1…](https://github.com/testdouble/standard/commit/479a048767b8bb69bf2da36c5541a5a3d31dd703)
- (9b5f084) [0.13.0](https://github.com/testdouble/standard/commit/9b5f084390b4f41a1b679ab58721283c8ec86f1d)


</details>